docs-v2/public/homepage/logo-figma.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let's maybe move these into an `icons` folder under `homepage/`? What do you think?

Reply:
Sounds good! Looks like we have an `icons` folder for all components. Do you recommend we put it there, or a new `icons` folder under `homepage`?

Reply:
`icons` folder for all components is a good idea - we can re-work these then to be React components themselves. Nicer inter-op with our existing Icon structures too. 


```
const FigmaLogo = ({ size = 24 }: Props) => ( <Pane is="svg" {...} /> )
```

for example